frontend_vue/src/components/ui/CardIncident.vue

```diff
@@ -3,6 +3,7 @@
     <button
       v-bind="$attrs"
       class="relative w-full px-16 py-8 transition duration-100 bg-white border text-grey-700 grouped group rounded-xl shadow-solid-shadow-grey border-grey-200 error-card"
+      :class="{ 'mb-8': lastKey }"
```

Comment:
I'm not 100% sure about this solution.
I would have wrapped CardIncident into a span element and add a class to the last of type

       ```
 <span
          v-for="(incident, index) in hitsList"
          :key="index"
          class="card-incident"
        >
          <CardIncident
              ......props etc
          ></CardIncident>
        </span>
```

and add a css
`.card-incident:last-of-type {
  margin-bottom: 8px;
}`

This seems more simple to me, but if you think yours works better please keep it

Reply:
For this one, I'd like to keep it `logical` vs `using CSS` because it is more scalable - _i.e. if we want to do more stuff to the last incident item in future_